pkg/controllers/user/pipeline/controller/pipelineexecution/deploy.go

```diff
@@ -806,6 +827,16 @@ func GetMinioDeployment(ns string) *appsv1beta2.Deployment {
 									ContainerPort: utils.MinioPort,
 								},
 							},
+							Resources: corev1.ResourceRequirements{
+								Limits: corev1.ResourceList{
+									corev1.ResourceCPU:    *resource.NewMilliQuantity(200, resource.DecimalSI),
+									corev1.ResourceMemory: *resource.NewQuantity(200E6, resource.BinarySI),
+								},
+								Requests: corev1.ResourceList{
+									corev1.ResourceCPU:    *resource.NewMilliQuantity(100, resource.DecimalSI),
+									corev1.ResourceMemory: *resource.NewQuantity(100E6, resource.BinarySI),
+								},
```

Comment:
I think hard code is not a good solution，if the project resource left is not enough to create these pods, they also can't work as expected. maybe make them configurable looks better.

Reply:
It should be configurable. But we also need a reasonable default so that these components have a higher QoS class. Burstable versus BestEffort in this case.